Makefile

```
@@ -285,6 +285,12 @@ docker-cilium-builder-manifest:
 	@$(ECHO_CHECK) contrib/scripts/push_manifest.sh cilium-builder $(UTC_DATE)
 	$(QUIET) contrib/scripts/push_manifest.sh cilium-builder $(UTC_DATE)
 
+docker-hubble-relay-image:
+	$(CONTAINER_ENGINE_FULL) build -f hubble-relay.Dockerfile -t "cilium/hubble-relay:$(DOCKER_IMAGE_TAG)" .
+	$(CONTAINER_ENGINE_FULL) tag cilium/hubble-relay:$(DOCKER_IMAGE_TAG) cilium/hubble-relay:$(DOCKER_IMAGE_TAG)-${GOARCH}
+	$(QUIET)echo "Push like this when ready:"
+	$(QUIET)echo "docker push cilium/hubble-relay:$(DOCKER_IMAGE_TAG)-${GOARCH}"
```

Nit: use `${CONTAINER_ENGINE}` instead of `docker` in the push instructions:

```suggestion
	$(QUIET)echo "
	${CONTAINER_ENGINE} push cilium/hubble-relay:$(DOCKER_IMAGE_TAG)-${GOARCH}"
```

I think this might also be wrong in some of the other `docker-*-image` targets where this was probably copied from.

---

It was indeed copied from the `docker-operator-image` target. I can update both targets if you want.

---

That'd be great, thanks.